src/System.Private.CoreLib/shared/System/Globalization/CultureInfo.cs

```diff
@@ -232,6 +227,20 @@ public CultureInfo(int culture, bool useUserOverride)
             _name = _cultureData.CultureName;
         }
 
+        private void InitializeFromName(string name, bool useUserOverride)
+        {
+            // Get our data providing record
+            _cultureData = CultureData.GetCultureData(name, useUserOverride);
+
+            if (_cultureData == null)
+            {
+                throw new CultureNotFoundException(nameof(name), name, SR.Argument_CultureNotSupported);
+            }
+
+            _name = _cultureData.CultureName;
+            _isInherited = (this.GetType() != typeof(System.Globalization.CultureInfo));
```

Comment:
The original code used `_isInherited = !this.EETypePtr.FastEquals(EETypePtr.EETypePtrOf<CultureInfo>());`. If the optimization should be kept I will have to separate it out from the shared code.

Reply:
This change is fine.